examples/1.Ducks/DetectorDuck/platformio.ini

```diff
@@ -19,8 +19,9 @@ monitor_speed = 115200
 monitor_filters = time
 
 lib_deps = 
-    ClusterDuck Protocol
-    ; /PATH-TO/ClusterDuck-Protocol
+    /Users/timowielink/Documents/Arduino/libraries/ClusterDuck-Protocol
+    /Users/timowielink/Documents/Arduino/libraries/ClusterDuck-Protocol/libraries
+    https://github.com/Project-Owl/Crypto.git
 
```

Comment:
Looks like you have some workstation-specific lines here @TimoWielink

Reply:
Let me change that!